system-tests/services/_scripts/external-volumes/teardown

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+pip install boto3
+pip install retrying
+
+cmd="./services/_scripts/external-volumes/volclean.py \
+-y -r $REGION -k $ACCESS_KEY_ID -s \"$SECRET_ACCESS_KEY\" \
+--ignore-metrics --tags 'Name:^integration-test-dcos-ui-$TEST_UUID'"
+
+MATCH_STRING="*Removing*Volumes*"
+TIMEOUT=30
+counter=0
+
+# This is the number of volumes we want to delete.
+# Check test-external-volumes.js to see how many volumes are created.
+# This in most cases corresponds to the number of test cases.
+num_vols_delete=2
+
+while [ "$num_vols_delete" -gt 0 ]
+do
+  echo "$num_vols_delete volume(s) left to delete"
+  while
+    result=$(eval "$cmd 2>&1")
+    echo $result
+    remove_string=$(echo $result | sed -n 's/^.*\(\Removing.*[0-9].*\Volumes\).*/\1/p')
+    num_removed=$(echo $remove_string | sed 's/[^0-9]*//g')
+    num_vols_delete=$(($num_vols_delete-${num_removed:-0}))
+    sleep 1s
```

Comment:
Is this an arbitrary amount of time to sleep? Maybe add a comment to explain why it's here.